src/client/Microsoft.Identity.Client/Platforms/Features/DefaultOSBrowser/HttpListenerInterceptor.cs

```diff
@@ -89,6 +89,13 @@ public HttpListenerInterceptor(ICoreLogger logger)
                 cancellationToken.ThrowIfCancellationRequested();
 
                 // if cancellation was not requested, propagate original ex
+                if (ex is HttpListenerException)
+                {
+                    throw new Exception("HttpListenerException occurred when listening for the response from the browser. " + 
+                        "One cause might be the app is unable to listen on the specified URL. " + 
+                        "One solution might be to run 'netsh http add iplisten 127.0.0.1' from the Admin command prompt.", 
```

Comment:
I would word it more like 

`An HttpListenerException occurred while listening on http:\\localhost:{port} for the system browser to complete the login. Possible cause and mitigation: .... "

Reply:
Done.